tests/v2/actions/kubeapi/configmaps/configmaps.go

```diff
@@ -0,0 +1,78 @@
+package configmaps
+
+import (
+	"context"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/unstructured"
+	"github.com/rancher/shepherd/pkg/api/scheme"
+	coreV1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// ConfigMapGroupVersionResource is the required Group Version Resource for accessing config maps in a cluster,
+// using the dynamic client.
+var ConfigMapGroupVersionResource = schema.GroupVersionResource{
+	Group:    "",
+	Version:  "v1",
+	Resource: "configmaps",
+}
+
+// NewConfigmapTemplate is a constructor that creates a configmap template
+func NewConfigmapTemplate(configmapName, namespace string, annotations, labels, data map[string]string) coreV1.ConfigMap {
+	if annotations == nil {
+		annotations = make(map[string]string)
+	}
+	if labels == nil {
+		labels = make(map[string]string)
+	}
+	if data == nil {
+		data = make(map[string]string)
+	}
+
+	return coreV1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        configmapName,
+			Namespace:   namespace,
+			Annotations: annotations,
+			Labels:      labels,
+		},
+		Data: data,
+	}
+}
+
+// CreateConfigMap is a helper function that uses the dynamic client to create a config map on a namespace for a specific cluster.
+// It registers a delete fuction.
+func CreateConfigMap(client *rancher.Client, clusterName, configMapName, description, namespace string, data, labels, annotations map[string]string) (*coreV1.ConfigMap, error) {
```

Comment:
I would think this could be an extension; perhaps it is too specific as-is though. What do you all think?

Reply:
1. It's too specific
2. It's really just converting a config map to an unstructured. We should really push people to use our controllers and new clients coming than continuing this pattern

Reply:
heard; I'm not actively using this package but I see what you're saying now. Resolved.

Reply:
too specific.